src/components/DiffView/index.tsx

```diff
@@ -53,6 +85,29 @@ export class DiffViewBase extends React.Component<Props> {
     }
   }
 
+  componentDidUpdate() {
+    this.loadData();
+  }
+
+  _loadData = () => {
+    const {
+      _fetchLinterMessages,
+      dispatch,
+      version,
+      linterMessages,
+      linterMessagesAreLoading,
+    } = this.props;
+
+    if (version && linterMessages === undefined && !linterMessagesAreLoading) {
```

Comment:
It seems the `version &&` is not tested.

```diff
diff --git a/src/components/DiffView/index.tsx b/src/components/DiffView/index.tsx
index ff5e4c1..25ee49c 100644
--- a/src/components/DiffView/index.tsx
+++ b/src/components/DiffView/index.tsx
@@ -98,7 +98,7 @@ export class DiffViewBase extends React.Component<Props> {
       linterMessagesAreLoading,
     } = this.props;
 
-    if (version && linterMessages === undefined && !linterMessagesAreLoading) {
+    if (/*version && */linterMessages === undefined && !linterMessagesAreLoading) {
       dispatch(
         _fetchLinterMessages({
           versionId: version.id,
```

Reply:
oh, whoops, it is not possible to render `DiffView` without a version now.